apps/total-typescript/src/pages/tutorials/[module]/[exercise]/index.tsx

```diff
@@ -37,8 +42,20 @@ export const getStaticPaths: GetStaticPaths = async (context) => {
   return {paths, fallback: 'blocking'}
 }
 
-const ExercisePage: React.FC<any> = ({exercise, module}) => {
-  return <ExerciseTemplate exercise={exercise} module={module} />
+const ExercisePage: React.FC<any> = ({
+  exercise,
+  module,
+  videoThumbId,
+  transcript,
+}) => {
+  return (
+    <ExerciseTemplate
+      exercise={exercise}
+      module={module}
+      videoThumbId={videoThumbId}
+      transcript={transcript}
```

Comment:
LessonResource also doesn't have a transcript but we want those at build time for seo etc

Reply:
the `exercise` is "cast down" in the `exercise-template` and "loses" data when it become a  `LessonResource` which is why we explicitly separate the transcript as a "media resource"